VideoAPI/VideoApi.Services/ConsultationService.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Extensions.Logging;
+using System;
+using VideoApi.Domain.Enums;
+using VideoApi.Services.Contracts;
+using VideoApi.Services.Kinly;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.Services
+{
+    public class ConsultationService : IConsultationService
+    {
+        private readonly IKinlyApiClient _kinlyApiClient;
+        private readonly ILogger<ConsultationService> _logger;
+
+        public ConsultationService(IKinlyApiClient kinlyApiClient, ILogger<ConsultationService> logger)
+        {
+            _kinlyApiClient = kinlyApiClient;
+            _logger = logger;
+        }
+
+
+        public Task TransferParticipantAsync(Guid conferenceId, Guid participantId, VirtualCourtRoomType fromRoom,
```

Comment:
I would suggest keeping to strings since the room names are dynamic

Reply:
updated to string